as/cNFT/assembly/models/log.ts

```diff
@@ -1,3 +1,11 @@
+/**
+ * **Event Log Classes**
+ *
+ * Classes used for capture data from the change methods in the contract when they are executed. The subgraph will capture these data and then saved it in the graph node.
```

Comment:
The Event Log is not used only by the subgraph, so I think is better to keep this speaking generally.